ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/139__trusted_ca_bundles.robot

```diff
@@ -85,8 +91,11 @@ Suite Teardown
 
 Restore DSCI Trusted CA Bundle Settings
     [Documentation]    Restore DSCI Trusted CA Bundle settings to original tate
+    [Arguments]    ${custsom_ca_value}
+
     Set Custom CA Bundle Value In DSCI    ${DSCI_NAME}   ''    ${OPERATOR_NS}
     Set Trusted CA Bundle Management State    ${DSCI_NAME}    Managed    ${OPERATOR_NS}
```

Comment:
The management state would also deserve similar check for original value.

Reply:
yes, though lets make this another PR since similar approach is needed for additional operator test cases, if that is OK with you.